### PR TITLE
create default root user for authentication (#16545)

### PR DESCRIPTION
**Pull request by jiangke-sr**
Signed-off-by: kejiang <ke.jiang@zilliz.com>

issue: https://github.com/milvus-io/milvus/issues/16545

ADD: create default root user when start up and cannot be deleted
ADD: password's length cannot less than 6 characters

### CODE REVIEW

File: internal/util/constant.go [hunk 1]
```diff
@@ -26,4 +26,6 @@ const (
 	// MemberCredID id for Milvus members (data/index/query node/coord component)
 	MemberCredID        = "@@milvus-member@@"
 	CredentialSeperator = ":"
+	UserRoot            = "root"
+	DefaultRootPassword = "123456"
```
Comment (jaime0815) [inline internal/util/constant.go:30]: use Milvus as a default password

**Comment by aakejiang** [comment]
rerun ut

File: internal/rootcoord/root_coord.go [hunk 2]
```diff
@@ -1119,6 +1122,13 @@ func (c *Core) Init() error {
 			c.impTaskKv,
 			c.CallImportService,
 		)
+		// init data
+		encryptedRootPassword, _ := crypto.PasswordEncrypt(util.DefaultRootPassword)
+		initError = c.MetaTable.AddCredential(&internalpb.CredentialInfo{Username: util.UserRoot, EncryptedPassword: encryptedRootPassword})
+		if initError != nil {
+			return
+		}
+		log.Debug("RootCoord init user root done")
```
Comment (xiaofan-luan) [inline internal/rootcoord/root_coord.go:1131]: the log of init function is a little bit messed up.

If initErr not nil, there should only one log about the error in WARN
if initErr not nil, there should be only one info log to show init successfully

File: internal/util/constant.go [hunk 3]
```diff
@@ -26,4 +26,6 @@ const (
 	// MemberCredID id for Milvus members (data/index/query node/coord component)
 	MemberCredID        = "@@milvus-member@@"
 	CredentialSeperator = ":"
+	UserRoot            = "root"
+	DefaultRootPassword = "Milvus"
```
Comment (xiaofan-luan) [inline internal/util/constant.go:30]: just a question, is Milvus password capital sensitive?

Comment (aakejiang) [reply internal/util/constant.go:30]: yes

File: internal/rootcoord/root_coord.go (hunk 2 above)
Comment (aakejiang) [reply internal/rootcoord/root_coord.go:1131]: another pr for this

**Pull request merged**